app/controllers/api/v3/converters_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
       #   "keyN": [ .. ]
       # }
       def stats
-        keys = params.require(:keys)
+        keys = permitted_params.to_h.fetch(:keys)
```

Comment:
The `#to_h` is needed or else it will throw this really long [Deprecation message in the logs](https://stackoverflow.com/questions/38710904/how-do-i-resolve-the-deprecation-warning-method-to-hash-is-deprecated-and-will).